resume/resume-2.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Nick Long Resume</title>
+	</head>
+	<body>
+		<style>
+	body {
+		background-color:rgb(243, 240, 234);
```

Comment:
i like this look!

Reply:
thanks, me too. I was thinking of Patrick Bateman (you know, American Psysho) looking over business cards